.github/workflows/upload_complete_workflow_metrics.py

```
@@ -0,0 +1,985 @@
+            print(error_msg)
+
+    def _upload_workflow_metrics(self, workflow_data: Dict[str, Any], jobs_data: Dict[str, Any]) -> None:
+        """Internal method to upload workflow metrics"""
+        db_data = {}
+        db_data[FIELD_ID] = f"github-workflow-{self.run_id}"
+        
+        
+        # Schema fields
+        # Use conclusion for completed workflows, fallback to status
+        db_data[FIELD_STATUS] = str(workflow_data.get('conclusion') or workflow_data.get('status', 'unknown'))
+        if db_data[FIELD_STATUS] is "success":
```

Ruff (0.13.2)

Use `==` to compare constant literals

Replace `is` with `==`

(F632)